doc/python/2d-histogram-contour.md

```diff
@@ -32,6 +32,19 @@ jupyter:
     permalink: python/2d-histogram-contour/
     thumbnail: thumbnail/hist2dcontour.png
 ---
+#### Contour Plot in Plotly Express
+
+[Plotly Express](/python/plotly-express/) is the easy-to-use, high-level interface to Plotly, which [operates on "tidy" data](/python/px-arguments/).
+In `px.density_contour` rows of data frame are grouped together into contour marks to visualize the 2D distribution of an aggregate function histfunc of the value z. To have a filled contour plot in Plotly express set [contours_coloring](https://plot.ly/python/reference/#contour-contours-coloring) in `update_trace` method to `fill`.
```

Comment:
This sentence is a bit hard to read. Maybe it could be:

The `px.density_contour` chart type groups rows of your data frame together into contour marks in order to visualize the 2D distribution of an aggregation function.